server/plugin/command/create_event.go

```diff
@@ -12,10 +13,15 @@ import (
 
 func getCreateEventFlagSet() *flag.FlagSet {
 	flagSet := flag.NewFlagSet("create", flag.ContinueOnError)
-	flagSet.String("subject", "", "Subject of the Event (no spaces for now)")
+	flagSet.Bool("help", false, "show help")
+	flagSet.String("subject", "", "Subject of the event (no spaces for now)")
+	flagSet.String("body", "", "Body of the event (no spaces for now)")
+	flagSet.StringSlice("location", []string{}, "Location of the event <displayName,street,city,state,postalcode,country> (comma separated; no spaces)")
 	flagSet.String("starttime", time.Now().Format(time.RFC3339), "Start time for the event")
+	flagSet.Bool("allday", false, "Set as all day event (starttime/endtime must be set to midnight on different days - 2019-12-19T00:00:00-00:00)")
+	flagSet.Int32("reminder", 15, "Reminder (in minutes)")
 	flagSet.String("endtime", time.Now().Add(time.Hour).Format(time.RFC3339), "End time for the event")
-	flagSet.StringSlice("attendees", []string{}, "A comma separated list of Attendees")
+	flagSet.StringSlice("attendees", []string{}, "A comma separated list of Mattermost UserIDs")
```

Comment:
1/5 I recommend accepting `@username`s, see https://github.com/levb/mattermost-plugin-solar-lottery/blob/a37a0ba0512bad58ced4a542740ec7cf9b53aa3c/server/plugin/api.go#L28 for an example.

Reply:
@levb, we need to continue the discussion on accessing the plugin API for this to happen.